tests/runtimes/grpc/geography.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+# SPDX-License-Identifier: Elastic-2.0
+
+from pathlib import Path
+
+from typegraph import Graph, Policy, typegraph
+from typegraph.runtimes.grpc import GrpcRuntime
+
+
+BASE_DIR = Path(__file__).parent
+
+
+@typegraph()
+def geography(g: Graph):
+    endpoint = "tcp://localhost:4770"
+
+    geography = BASE_DIR.joinpath("proto/geography.proto")
```

Comment:
Paths in a typegraph definition are always relative to the parent directory. You could resolve the absolute path in the typegraph core.